.glotter.yml

```diff
@@ -8,286 +8,295 @@ projects:
+                        - "    *************"
+                        - "   ***************"
+                        - "  *****************"
+                        - " *******************"
+                        - "*********************"
+                        - " *******************"
+                        - "  *****************"
+                        - "   ***************"
+                        - "    *************"
+                        - "     ***********"
+                        - "      *********"
+                        - "       *******"
+                        - "        *****"
+                        - "         ***"
+                        - "          *"
+                transformations:
+                    - "splitlines"
     binarysearch:
         words:
             - "binary"
             - "search"
         requires_parameters: true
         tests:
             binary_search_valid:
-                -   name: "sample input first true"
-                    input: '"1, 3, 5, 7" "1"'
-                    expected: "true"
-                -   name: "sample input last true"
-                    input: '"1, 3, 5, 7" "7"'
-                    expected: "true"
-                -   name: "sample input middle true"
-                    input: '"1, 3, 5, 7" "5"'
-                    expected: "true"
-                -   name: "sample input one true"
-                    input: '"5" "5"'
-                    expected: "true"
-                -   name: "sample input one false"
-                    input: '"5" "7"'
-                    expected: "false"
-                -   name: "sample input many false"
-                    input: '"1, 3, 5, 6" "7"'
-                    expected: "false"
-                -   name: "sample input middle true"
-                    input: '"1, 2, 3, 4, 5, 6, 7" "3"'
-                    expected: "true"
+                cases:
+                    -   name: "sample input first true"
+                        input: '"1, 3, 5, 7" "1"'
+                        expected: "true"
+                    -   name: "sample input last true"
+                        input: '"1, 3, 5, 7" "7"'
+                        expected: "true"
+                    -   name: "sample input middle true"
+                        input: '"1, 3, 5, 7" "5"'
+                        expected: "true"
+                    -   name: "sample input one true"
+                        input: '"5" "5"'
+                        expected: "true"
+                    -   name: "sample input one false"
+                        input: '"5" "7"'
+                        expected: "false"
+                    -   name: "sample input many false"
+                        input: '"1, 3, 5, 6" "7"'
+                        expected: "false"
+                    -   name: "sample input middle true"
+                        input: '"1, 2, 3, 4, 5, 6, 7" "3"'
+                        expected: "true"
+                transformations:
```

Comment:
Love this! Very cool feature. Allows this to be expanded, so we don't worry about missing the old python setup.